packages/reports/src/Components/Routes/binary-routes.jsx

```diff
@@ -19,8 +19,8 @@ const BinaryRoutes = props => {
             }}
         >
             <Switch>
-                {getRoutesConfig({ is_appstore }).map(route => (
-                    <RouteWithSubRoutes key={route.path} {...route} {...props} />
+                {getRoutesConfig({ is_appstore }).map((route, i) => (
+                    <RouteWithSubRoutes key={i} {...route} {...props} />
```

Comment:
eliminating a console warning: _Each child in a list should have a unique "key" prop. Check the render method of `BinaryRoutes`_